ui/eureka/eureka.js

```diff
@@ -2723,7 +2723,7 @@ class EurekaTracker {
         }
       }
 
-      if (!this.zoneInfo.hasTracker)
+      if (!this.zoneInfo.hasTracker && this.zoneInfo.hasTracker !== undefined)
```

Comment:
Oh oops, sorry for not catching this.  Could you leave this conditional the same, and add `hasTracker: true` to the four Eureka zones instead?